src/frontegg-oauth-client.ts

```diff
@@ -7,6 +7,7 @@ export interface FronteggUserData {
   email: string
   profilePictureUrl: string | null | undefined
   externalWorkspaceId: string
+  impersonated: boolean
```

Comment:
🟢 Personal preference - I would prefix it with `is` to make it obvious that it is boolean.